samples/python/hello_classification/hello_classification.py

```diff
@@ -34,70 +36,80 @@ def main():
+    # - apply linear resize from tensor spatial dims to model spatial dims
+    preproc.input().preprocess().resize(ResizeAlgorithm.RESIZE_LINEAR)
+    # 3) Here we suppose model has 'NCHW' layout for input
+    preproc.input().network().set_layout(Layout("NCHW"))
+    # 4) Set output tensor information:
+    # - precision of tensor is supposed to be 'f32'
+    preproc.output().tensor().set_element_type(Type.f32)
+    # 5) Apply preprocessing modifing the original 'model'
+    model = preproc.build()
 
     # Get a number of classes recognized by a model
-    num_of_classes = max(net.outputs[out_blob].shape)
+    num_of_classes = max(model.output().shape)
 
-    # ---------------------------Step 4. Loading model to the device-------------------------------------------------------
+    # ---------------------------Step 5. Loading model to the device-------------------------------------------------------
     log.info('Loading the model to the plugin')
-    exec_net = ie.load_network(network=net, device_name=args.device)
-
-    # ---------------------------Step 5. Create infer request--------------------------------------------------------------
-    # load_network() method of the IECore class with a specified number of requests (default 1) returns an ExecutableNetwork
-    # instance which stores infer requests. So you already created Infer requests in the previous step.
-
-    # ---------------------------Step 6. Prepare input---------------------------------------------------------------------
-    original_image = cv2.imread(args.input)
-    image = original_image.copy()
-    _, _, h, w = net.input_info[input_blob].input_data.shape
-
-    if image.shape[:-1] != (h, w):
-        log.warning(f'Image {args.input} is resized from {image.shape[:-1]} to {(h, w)}')
-        image = cv2.resize(image, (w, h))
-
-    # Change data layout from HWC to CHW
-    image = image.transpose((2, 0, 1))
-    # Add N dimension to transform to NCHW
-    image = np.expand_dims(image, axis=0)
-
-    # ---------------------------Step 7. Do inference----------------------------------------------------------------------
+    exec_net = core.compile_model(model=model, device_name=args.device)
+    
+    # ---------------------------Step 6. Create infer request--------------------------------------------------------------
     log.info('Starting inference in synchronous mode')
-    res = exec_net.infer(inputs={input_blob: image})
+    
+    res = exec_net.infer_new_request(inputs={0:image_tensor})
 
-    # ---------------------------Step 8. Process output--------------------------------------------------------------------
+    # ---------------------------Step 7. Process output--------------------------------------------------------------------
     # Generate a label list
     if args.labels:
         with open(args.labels, 'r') as f:
             labels = [line.split(',')[0].strip() for line in f]
 
-    res = res[out_blob]
+    predictions = next(iter(np.array(res)))
```

Comment:
now infer result is dict Node is key, data is value (np.array)
```suggestion
    predictions = next(iter(res.values))
```